test/unit/init/init_test.js

```diff
@@ -87,6 +84,14 @@ suite('init', () => {
       });
     });
 
+    test('works with default env', () => {
```

Comment:
this is testing the "not found" case as much as it's testing that it doesn't throw with a default env. Would be worth mentioning in the test description